megamek/src/megamek/common/BombType.java

```diff
@@ -394,11 +420,67 @@ private static BombType createClusterBomb() {
 		        .setISApproximate(false, false, false, false, false)
 		        .setClanAdvancement(DATE_PS, DATE_PS, DATE_PS, DATE_NONE, DATE_NONE)
 		        .setClanApproximate(false, false, false, false, false);
+		
+		blastRadius.put(BombType.getBombInternalName(BombType.B_CLUSTER), 1);
 
 		return bomb;
 	}
 
-	// TODO Fuel-Air Bombs (See IO 165)
+	private static BombType createSmallFuelAirBomb() {
+        BombType bomb = new BombType();
+
+        bomb.name = "Fuel-Air Bomb (Small)";
+        bomb.shortName = "FAEBomb_s";
```

Comment:
shortName is used for display to users in contexts where the full name may be too long. It's used particularly on record sheets.